README.md

```diff
@@ -1 +1,89 @@
-# cesium-workshop
\ No newline at end of file
+# cesium-workshop
+
+<p align="center">
+    <a href="http://cesiumjs.org/">
+        <img src="https://github.com/AnalyticalGraphicsInc/cesium/wiki/logos/Cesium_Logo_Color.jpg" width="50%" />
+    </a>
+</p>
+
+A simple JavaScript app showcasing some features of [Cesium](http://cesium.agi.com/), the open-source WebGL virtual globe and map engine.  Just fork this repo and start coding.
+
+**Cesium version**: [1.34](http://cesiumjs.org/downloads.html).
```

Comment:
Should this be Cesium 1.35? Or probably Cesium 1.36 because the conference is in August?